adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -300,13 +331,46 @@ private async void AcquireTokenSilentWithBroker()
             {
                 Device.BeginInvokeOnMainThread(() =>
                 {
+                    this.testResult.Text = string.IsNullOrWhiteSpace(accessToken) ? "Result: Failure" : "Result: Success";
                     this.result.Text += "Result : " + output;
 
                     this.result.Text += "Logs : " + DrainLogs();
                 });
             }
         }
 
+        private void AcquireTokenWithBrokerButton_Clicked(object sender, EventArgs e)
+        {
+            switch (Device.RuntimePlatform)
+            {
+                case Device.iOS:
+                    AcquireTokenWithBroker();
```

Comment:
you can fall through the android case, see https://stackoverflow.com/questions/174155/switch-statement-fallthrough-in-c